pulsar/schema.go

```diff
@@ -124,21 +124,10 @@ func NewSchema(schemaType SchemaType, schemaData []byte, properties map[string]s
 	return
 }
 
-type AvroCodec struct {
-	Codec *goavro.Codec
-}
-
-func NewSchemaDefinition(schema *goavro.Codec) *AvroCodec {
```

Comment:
Even though the client is still in the development version (0.x.y), it is widely used in production. We should avoid introducing breaking changes in the v0.x version.

We may still need to support goavro here but marked it with deprecation. We can make these breaking changes and remove it in v1.0.

Reply:
Ok. Now I'm +1.